orttraining/orttraining/python/ort_trainer.py

```diff
@@ -660,16 +663,38 @@ def __init__(self, model, loss_fn, model_desc, training_optimizer_name, map_opti
         self._use_deterministic_compute = _use_deterministic_compute
         self.use_invertible_layernorm_grad = use_invertible_layernorm_grad
 
+        self._debug_conversion = _debug_conversion
+
         # use this special string to workaround a corner case that external loss_scale is passed into train_step as kwargs.
         # see prepare_input_and_fetches for more details.
         self.loss_scale_input_name = 'default_loss_scale_input_name'
 
         self._init_session()
 
+
+    def _debug_init_session(self):
+        self.no_postprocess_onnx_model = None
+        if not self._enable_internal_postprocess or not self._extra_postprocess:
+            self.no_postprocess_onnx_model = self.onnx_model_
+                
+            self.no_postprocess_session, self.no_postprocess_train_io_binding, self.no_postprocess_eval_io_binding, self.no_postprocess_output_name, _, self.no_postprocess_output_types = \
```

Comment:
all the new class properties that are being created are private, so it is better to preprend an underscore on them (e.g. `self._no_postprocess_session`, etc